[2025-01-06T19:27:42.255+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-06T19:27:42.296+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_dag.start_task manual__2025-01-06T19:27:38.282634+00:00 [queued]>
[2025-01-06T19:27:42.317+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_dag.start_task manual__2025-01-06T19:27:38.282634+00:00 [queued]>
[2025-01-06T19:27:42.318+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-06T19:27:42.352+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): start_task> on 2025-01-06 19:27:38.282634+00:00
[2025-01-06T19:27:42.364+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=120) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-06T19:27:42.367+0000] {standard_task_runner.py:72} INFO - Started process 121 to run task
[2025-01-06T19:27:42.369+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'kaggle_dag', 'start_task', 'manual__2025-01-06T19:27:38.282634+00:00', '--job-id', '866', '--raw', '--subdir', 'DAGS_FOLDER/kaggle_dag.py', '--cfg-path', '/tmp/tmp9uqc40rg']
[2025-01-06T19:27:42.372+0000] {standard_task_runner.py:105} INFO - Job 866: Subtask start_task
[2025-01-06T19:27:42.469+0000] {task_command.py:467} INFO - Running <TaskInstance: kaggle_dag.start_task manual__2025-01-06T19:27:38.282634+00:00 [running]> on host 9624078adb21
[2025-01-06T19:27:42.703+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_dag' AIRFLOW_CTX_TASK_ID='start_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-06T19:27:38.282634+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-06T19:27:38.282634+00:00'
[2025-01-06T19:27:42.706+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-06T19:27:42.794+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-06T19:27:42.796+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Here we start! "']
[2025-01-06T19:27:42.820+0000] {subprocess.py:99} INFO - Output:
[2025-01-06T19:27:42.824+0000] {subprocess.py:106} INFO - Here we start!
[2025-01-06T19:27:42.825+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-06T19:27:42.881+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-06T19:27:42.882+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=kaggle_dag, task_id=start_task, run_id=manual__2025-01-06T19:27:38.282634+00:00, execution_date=20250106T192738, start_date=20250106T192742, end_date=20250106T192742
[2025-01-06T19:27:42.951+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-06T19:27:43.015+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-06T19:27:43.028+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
